README.md

```diff
@@ -103,6 +103,39 @@ The solutions contain single page applications (SPAs) and use `IIS Express`. To
   - Ensure all HTML template files are closed when the application is run, or:
   - Manually change the browser address to the root URL.
 
+## Preparing your machine for advanced exercises
+
+If you completed all the standard exercises, first of all, well done!
+
+To complete the advanced exercises, you'll need to install the rest of the Particular Service Platform:
+
+- ServicePulse: a web app which monitors your endpoints.
+- ServiceInsight: a desktop app which visualises the flow of messages within and between endpoints in detail.
+- ServiceControl: a Windows service which collects the data that drives ServicePulse and ServiceInsight.
+
+Run the [Particular Platform Installer](https://particular.net/start-platform-download) again, selecting ServiceControl, ServiceInsight, and ServicePulse.
+
+When installation has completed, click "Start ServiceControl Management".
+
+You may now close the Particular Service Platform Installation window.
+
+In the ServiceControl Management window, click "Add new instance".
+
+In the settings for the new instance, there are two choices you must make:
+
+- "Transport": choose "MSMQ".
+- "Audit forwarding": choose "Off".
+
+All other settings should be left with their default values.
+
+Click "Add".
+
+After the instance has been added, make a note of the name of the instance. This will usually be "Particular.ServiceControl".
```

Comment:
From [the docs](https://docs.particular.net/servicecontrol/plugins/saga-audit#configuration-servicecontrol-queue):

> The queue name is based on the ServiceControl instance name...

which means you have to configure it like so:

```C#
var endpointConfiguration = new EndpointConfiguration("myendpoint");
endpointConfiguration.SagaPlugin("Particular.ServiceControl");
```